demo/web/index.html

```diff
@@ -185,31 +420,90 @@ <h1>Orca Web Demo</h1>
+  <button id="choose-single-btn">
+    Single Synthesize
+  </button>
+  <p><b>OR</b></p>
+  <label for="choose-stream-btn">Synthesize speech live as you type:</label>
+  <button id="choose-stream-btn">
+    Streaming Synthesis
+  </button>
+  <br />
+  <hr />
+</div>
+<div id="single-synthesis" style="display: none">
+  <label for="text-to-synthesize">Text to synthesize</label>
+  <br />
+  <textarea id="text-to-synthesize" rows="5" cols="50">
 </textarea>
-<div id="text-to-synthesize-error">&nbsp;</div>
-<label>
-  Speech Rate: <span id="speech-rate-display">1.0</span>
-  <input
-    id="speech-rate"
-    type="range"
-    min="0.7"
-    max="1.3"
-    value="1"
-    step="0.1"
-    disabled
-  />
-</label>
-<br />
-<hr />
-<button id="synthesize-btn" onclick="synthesize()" disabled>
-  Synthesize
-</button>
-<button id="control-btn" disabled>Play</button>
-<button id="download-btn" disabled onclick="downloadDumpAudio()">Download Audio</button>
-<br />
+  <div><span id="text-to-synthesize-num-chars">0</span>/<span id="max-char-limit"></span></div>
+  <div id="text-to-synthesize-error">&nbsp;</div>
+  <label>
+    Speech Rate: <span id="speech-rate-display">1.0</span>
+    <input
+      id="speech-rate"
+      type="range"
+      min="0.7"
+      max="1.3"
+      value="1"
+      step="0.1"
+    />
+  </label>
+  <button id="synthesize-btn">Synthesize</button>
+  <br />
+  <hr />
+  <button id="control-btn" disabled>Play</button>
+  <button id="download-btn" onclick="downloadDumpAudio()" disabled>Download Audio</button>
+  <br />
+  <br />
+  <table id="alignments-table" style="display: none; max-height: 20rem; overflow-y: auto; width: fit-content">
+    <colgroup>
+      <col span="1" style="width: 20%" />
+      <col span="1" style="width: 20%" />
+      <col span="1" style="width: 20%" />
+      <col span="1" style="width: 40%" />
+    </colgroup>
+    <tr>
+      <th style="text-align: left">Word</th>
+      <th style="text-align: left">Start time (s)</th>
+      <th style="text-align: left">End time (s)</th>
+      <th style="text-align: left">Phonemes</th>
+    </tr>
+  </table>
+</div>
+<div id="stream-synthesis" style="display: none">
+  <label>
+    Speech Rate: <span id="stream-speech-rate-display">1.0</span>
+    <input
+      id="stream-speech-rate"
+      type="range"
+      min="0.7"
+      max="1.3"
+      value="1"
+      step="0.1"
+    />
+  </label>
+  <br />
+  <br />
+  <button id="stream-open-btn">Open Stream</button>
+  <br />
+  <br />
+  <label for="stream-text-to-synthesize" style="font-weight: bold">Text to synthesize</label>
+  <br />
+  <input id="stream-text-to-synthesize" disabled />
+  <br />
+  <br />
+  <div style="font-weight: bold">Text passed to Orca Stream:</div>
+  <div id="stream-text-to-synthesize-display"></div>
+  <div id="stream-text-to-synthesize-error">&nbsp;</div>
+  <br />
+  <button id="stream-flush-btn" disabled>Flush</button>
+  <button id="stream-close-btn" disabled>Close</button>
```

Comment:
`Close Stream` ?

Reply:
Also noticed that closing and opening a stream does not clear the text under `Text passed to Orca Stream`